### ISSUE TITLE
Which Node.js LTS versions are supported by Tessel 2?

### ISSUE THREAD

**Issue by bennycode** [tessel/t2-start#156]
I checked the documentation on http://tessel.github.io/t2-start/ and it says that:
> Tessel currently supports the Long Term Support (LTS) version of Node.js.

As of today (https://github.com/nodejs/LTS#lts-schedule) there are multiple LTS versions of Node.js (v4.x & v6.x), so does Tessel 2 supports all of them?

**Comment by rwaldron** [comment]
Presently Node.js 4.5.0, due to [issues compiling Node.js 6.x.x](https://github.com/tessel/openwrt-tessel/pull/68)

**Comment by RTongue** [comment]
Has this changed since last year? Just got a new t2 and after updating the firmware it shows Node.js 6.10.3.

**Comment by HipsterBrown** [comment]
@RTongue Yes, our latest OS image for Tessel includes Node 6.10.3. We're now working on Node 8 support since it is the latest LTS.

**Comment by danawoodman** [comment]
When will v6.x be supported?

**Comment by Frijol** [comment]
@danawoodman it already is!

Output from `t2 version`:
```
INFO Tessel Environment Versions:
INFO t2-cli: 0.1.14
INFO t2-firmware: 0.1.0
INFO Node.js: 6.10.3
```

**Comment by danawoodman** [comment]
@Frijol sorry I mean v8.x 😆

**Comment by rwaldron** [comment]
@danawoodman we're behind because the t2-release scripts need to be serviced and no one has had time or understanding of the system to do the work to fix them :( Additionally, we'd need a PR to https://github.com/tessel/openwrt-tessel/pulls that has upgrades node.js, similar to this changeset: https://github.com/tessel/openwrt-tessel/pull/73/files#diff-0564297a9266694528aabbd6408d1075

**Comment by danawoodman** [comment]
Who has done the previous upgrades? Keeping Tessel supporting the latest node would be a huge win 👍

**Comment by rwaldron** [comment]
I did the last one, but had to do every step manually because our release script used modules that were no longer compatible with some AWS API.
